yelp_scrape.py

```diff
@@ -63,12 +64,40 @@ def collect_webpages(soup: BeautifulSoup, search_item: str) -> set:
     biz_lst: list = []
     for a in soup.find_all('a'):
         if 'href' in a.attrs:
-            if a.attrs['href'].split('?')[-1] == 'osq=' + search_item:
+            if a.attrs['href'].split('?')[-1] == 'osq=' + search_item and a.attrs['href'] not in biz_lst:
                 biz_lst.append(a.attrs['href'])
-    return set(biz_lst)
-
-
+        if len(biz_lst) >= 5:
+            break
+    return biz_lst
 
+def page_info_grab(pages: list) -> dict:
```

Comment:
I'd put more information in the type header
`list` and `dict` are collection types, it's important to know what they contain
you can specify their subtypes like `list[str]` and `dict[str, int]` (dict has two types, the first is the type of they keys, the second is the type of the values)